src/Microsoft.AspNetCore.BuildTools.Tasks/Microsoft.AspNetCore.BuildTools.Tasks.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Yup. The intent being we really don't want folks to assume these are general purpose build tools that Microsoft is shipping.